msdk-io/msdk-io-mzml/src/test/java/io/github/msdk/io/mzml/MzMLFileExportMethodTest.java

```diff
@@ -246,7 +246,7 @@ public void testFileWithUV() throws MSDKException, IOException {
         scan.getNumberOfDataPoints());
     Assert.assertEquals(Integer.valueOf(2114), scan.getScanNumber());
     Assert.assertEquals(MsSpectrumType.CENTROIDED, scan.getSpectrumType());
-    Assert.assertEquals(9.939699e06f, scan.getTIC(), 0.0001);
+    Assert.assertEquals(9.939699e06f, scan.getTIC(), 1e01);
```

Comment:
OK, I guess it is not so surprising to get this kind of error when using floats. But the issue of the mean filter module should be something else.